src/lib/createVNode.js

```diff
@@ -1,3 +1,14 @@
+/**
+ * VirtualNode를 생성하는 함수
+ * @param {string} type - 노드의 타입
+ * @param {Object} props - 노드의 속성
+ * @param {...any} children - 노드의 자식 노드
+ * @returns {Object} 가상돔 노드
+ */
 export function createVNode(type, props, ...children) {
-  return {};
+  return {
+    type,
+    props,
+    children: [...children].flat(Infinity).filter((child) => child !== null && child !== undefined && child !== false),
```

Comment:
```suggestion
    children.flat(Infinity).filter((x) => x || x === 0),
```
이 구문을 이렇게 바꿔보시는 건 어떠실지 제안 드립니다!

Reply:
flat과 filter는 새로운 배열을 반환하는 함수이기도하고 children의 depth가 깊기 때문에 얕은 복사를 안해도 될것 같아용!

Reply:
오 훨씬 깔끔해지네요 감사합니다!